### DISCUSSION TITLE
Filtering a product collection by sale / how to see if a product is on sale

### DISCUSSION THREAD

**Discussion by austriker27** [statamic-rad-pack/shopify#47] [Help]
I'm pretty new to shopify so I don't know if this is possible, but is it possible to get items that are on sale? I tried adding an item to shopify that's on sale, importing it specifically into statamic (and updated the cache) but I dont see any values to indicate its on sale in the CMS. Did I do this wrong? 

Something being able to use the statamic filter in a collection would be ideal I think! 

```
{{ collection:products limit="12"  filter="saleIsCurrent" }}

            {{ partial:components/product_card }}

{{ /collection:products }}
```

If this isn't possible, is it possible to maybe we could use a conditional toggle field on the products that checks the "compare at" price from Shopify versus regular price? 

Related: I could see how sorting by products on sale would also be useful.

**Comment by jackabox** [comment]
There isn't anything for the sale at the moment, this is a bit of an oversight on my part, the data should be easy enough to fetch though - create an issue for this and I'll get it actioned when I jump on to do some updates.

As for the filter that'd be a great FR also.

**Reply by austriker27** [reply]
Sounds good. I'll create FR issues for both!

**Comment by ryanmitchell** [comment]
@austriker27 I've added a PR here for this, and docs to describe its usage: https://github.com/statamic-rad-pack/shopify/pull/180

**Reply by austriker27** [reply]
Oh nice! I'm no longer using this addon but I bet others will find this useful!
